public/app/features/browse-dashboards/components/BrowseActions/DescendantCount.tsx

```diff
@@ -14,24 +13,16 @@ export interface Props {
 }
 
 export const DescendantCount = ({ selectedItems }: Props) => {
-  const styles = useStyles2(getStyles);
   const { data, isFetching, isLoading, error } = useGetAffectedItemsQuery(selectedItems);
 
   return (
-    <div className={styles.breakdown}>
-      <>
-        {data && buildBreakdownString(data.folder, data.dashboard, data.libraryPanel, data.alertRule)}
-        {(isFetching || isLoading) && <Spinner size={12} />}
-        {error && <Alert severity="error" title="Unable to retrieve descendant information" />}
-      </>
-    </div>
+    <>
+      {data && (
+        <P color="secondary">{buildBreakdownString(data.folder, data.dashboard, data.libraryPanel, data.alertRule)}</P>
```

Comment:
the different font size here causes some content shift when combined with the spinner

could just up the spinner size to 14, or we could maybe do something cleverer with getting the value from the theme? 🤔

Reply:
great catch!